### PR TITLE
Improve view for companies

### PR DESCRIPTION
**Pull request by IvanDanyliv**
[Trello Task](https://trello.com/_____)

**CHECK LIST**
- [x] specs added
- [x] specs passed
- [x] code coverage >90%
- [x] DB indexes added if needed
- [x] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [x] "let!" is used in specs
- [x] screenshot of feature is added if task has UI

image
image
image
image

### CODE REVIEW

File: config/locales/ua.yml [hunk 1]
```diff
@@ -289,9 +290,10 @@ ua:
       companies_title: 'Всі компанії'
 
     search_form:
-      name_email_phone_placeholder: 'Назва, електронна адреса чи номер телефону компанії'
-      company_type_prompt: 'Виберіть тип компанії'
+      name_email_phone_placeholder: 'Пошук'
+      company_type_prompt: 'Виберіть                                                                                                                                                                  '
```
Comment (mmarusyk) [inline config/locales/ua.yml:294]: space

**Review by mmarusyk — APPROVED** [review]

**Review by matsevich — APPROVED** [review]

**Review by Andrii-Kozak — APPROVED** [review]

File: app/javascript/stylesheets/admin_all_component.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  padding-top: 20px;
+  padding-bottom: 20px;
+  .all-component{
+    background-color: rgba(255, 255, 255, 0.4);
+    border-radius: 4px 4px 0 0;
+  }
+
+  nav.navbar.navbar-dark {
+    background-color:  rgba(18, 19, 19, 0.616);
+    border-radius: 4px 4px 0 0;
```
Comment (pereslop) [inline app/javascript/stylesheets/admin_all_component.scss:14]: duplicated property, move it to varaible or DRY it

**Review by DenisRem — APPROVED** [review]

File: app/javascript/stylesheets/admin_all_component.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      align-items: center;
+    } 
+
+    td.eye-btn {
+      text-align: center
+    }
+
+    th.plus-btn{
+      text-align: center
+    }
```
Comment (pereslop) [inline app/javascript/stylesheets/admin_all_component.scss:36]: DRY it

File: app/javascript/stylesheets/admin_form.scss [hunk 4]
```diff
@@ -0,0 +1,49 @@
+.form-component {
+  width: 50%;
+  display: block;
+  margin: auto;
+  padding: 20px;
+  background-color: rgba(255, 255, 255, 0.4);
```
Comment (pereslop) [inline app/javascript/stylesheets/admin_form.scss:6]: duplicated color, move common colors to https://github.com/IF-115-Ruby/OSBB/blob/development/app/javascript/stylesheets/_colors.scss, and reuse it

File: app/javascript/stylesheets/admin_form.scss [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  border-radius: 10px;
+  min-height: 36px;
+  ul {
+    padding: 0;
+    li {
+      margin: 5px 5px;
```
Comment (pereslop) [inline app/javascript/stylesheets/admin_form.scss:12]: margin: 5px; should be enought

File: app/views/account/admin/companies/index.html.slim [hunk 6]
```diff
@@ -1,13 +1,29 @@
...
-  .pagination-container
-    = paginate @companies, theme: 'twitter-bootstrap-4',
-                       pagination_class: "pagination-md"
+.box
+  .all-component
+    nav.navbar.navbar-dark
+      h2 
+        = t('companies.index.companies_title')
+      .form-row
+        =render 'search_form'
```
Comment (pereslop) [inline app/views/account/admin/companies/index.html.slim:7]: add space after `=`

File: app/javascript/stylesheets/admin_all_component.scss [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+  nav.navbar.navbar-dark {
+    background-color:  rgba(18, 19, 19, 0.616);
+    border-radius: 4px 4px 0 0;
+    color: white;
+  }
+
+  table.table {
+    thead.thead {
+      background-color:  rgba(18, 19, 19, 0.616);
```
Comment (pereslop) [inline app/javascript/stylesheets/admin_all_component.scss:20]: two spaces after `background-color:`

File: app/javascript/stylesheets/_colors.scss [hunk 8]
```diff
@@ -3,3 +3,6 @@ $text-raisin: #262626;
 $light-steel-blue: #448ab5a3;
 $gainsboro: #e6e6e68f;
 $solitude: #f8f9fa6e;
+$opasity-gray:rgba(255, 255, 255, 0.4);
```
Comment (chu4) [inline app/javascript/stylesheets/_colors.scss:6]: space after :

**Review by chu4 — APPROVED** [review]

**Pull request merged**